src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        return randomColor;
```

Comment:
According to the checklist, you should avoid creating temporary variables if you can directly return the value from the method. Instead of creating a 'randomColor' variable, you can directly return the result of 'colorSupplier.getRandomColor()'.